packages/devextreme-scss/scss/widgets/generic/chat/_colors.scss

```diff
@@ -76,6 +96,11 @@ $chat-errorlist-color: $base-invalid-color !default;
   $chat-messagelist-empty-icon-background-color: $base-inverted-bg !default;
   $chat-messagelist-empty-message-color: $base-text-color !default;
   $chat-messagelist-empty-prompt-color: $base-text-color !default;
+
+  $chat-typingindicator-color: $chat-information-color !default;
+  $chat-typingindicator-bubble-bg-color: $chat-bubble-background-color-secondary !default;
+  $chat-typingindicator-circle-bg-color: lighten($base-bg, 32.55) !default;
```

Comment:
Kindly to be sure that all colors are adjusted according to correspond color themes.
<img width="461" alt="image" src="https://github.com/user-attachments/assets/ae096360-b01a-4da9-bfe6-8dc3b16044a0">
The main colors that are using are white, black, accent. Don't forget to get designer approve for using these colors.